### DISCUSSION TITLE
Missing Parts for StormSim

### DISCUSSION THREAD

**Discussion by fgarcia0128** [coastal-hazards-system/StormSim-Library#23] [General]
1.  Custom modeling integration into CHS computational framework. (With project conditions)
- PMs
- Storm selection 
- CRL's ( Location of PM as stablished by Norberto ). How do we assign this to all SP in project reach?
- HC are not matching 
2.  Seasonality and more resolution in time for MCS
3. Damage Equations -> (Jeff still working on it)
 - mean damage curve (for all LC) 
 - 1 standard deviation curve (for all LC)
 -  Show cloud of data points 
4. LCS code
 - Current temporal resolution -> Yearly 
 - There is some code for doing monthly sampling (Jeff)
 5. HC confidence limit issues
 - SST tight CL are to close the BE curve
 - Revisit CC Confidence limits crossover for combine storms
 6. Dn50 Seaside & Leeside 
 -Going to use Dn50, delta & water density (could also use S.G.)
7. Have Kevin and Andrew look at inputs and unit nomenclature
-Switch to imperial as default 
-Add switch for getting outputs on SI
8. Dn50 Calculations
-RB3 + LCS hybrid (Keep as separate kernels , like its now)
9. Berms  (low priority)
- Berm recession (TBD physics) -> Mobile berm 
- Static berm just requires to specify the slope of interest 
10. Stability for ends of breakwaters

**Comment by KHodgens** [comment]
11. CHS Storm subselection: tool to generate a suite of events that will adequately cover probability space for a CSRM study area given the number of lifecycles anticipated and the duration of lifecycles.
12. Once MCS is reconfigured to distribute storms to a particular day, need a transfer function to alter the hydrograph based on the date and SLC, seasonality, and tides.  
13.  Integrate q through time to deliver Q [VOL/DISTANCE].  Develop hazard curve for Q (a consistent StormSim output with q).  Serve Q to downstream processes (see 14 below). How should uncertainty be handled here for good confidence limits?
14.  CHART helper function: integrate Q [VOL/DISTANCE] across representative length to provide total flood volume predicted to overtop structure.  This will be compared with stage-volume relationship in a downstream process to ultimately be used in consequence calcs.
15.   Boundary Condition preparations: translate waves and water levels from (12 above) into BC's for morphology model.  Consider if Morphology model outputs can provide enhanced hydro and/or geometry that would be superior for overtopping calculations.  Also consider the potential for other downstream programs (AdH, RAS, CSTORM).
